server/utils/applications/utils.test.ts

```diff
@@ -594,22 +596,36 @@ describe('utils', () => {
       expect(isInapplicable(application)).toEqual(false)
     })
 
-    it('should return true if the applicant has answered no to the shouldPersonBePlacedInMaleAp question', () => {
-      mockOptionalQuestionResponse({ shouldPersonBePlacedInMaleAp: 'no' })
+    it('should return false if the applicant has not answered the isExceptionalCase question', () => {
+      mockOptionalQuestionResponse({})
 
-      expect(isInapplicable(application)).toEqual(true)
+      expect(isInapplicable(application)).toEqual(false)
     })
+  })
 
-    it('should return false if the applicant has answered yes to the shouldPersonBePlacedInMaleAp question', () => {
-      mockOptionalQuestionResponse({ shouldPersonBePlacedInMaleAp: 'yes' })
+  describe('isWomensAp', () => {
+    it(`should return true if the person from nDelius has a sex of 'Male'`, () => {
```

Comment:
Test title seems the wrong way round here!

Also, given we rely on the answer to `shouldPersonBePlacedInMaleAp` to determine the output on this, it would be nice to be explicit and include it in the test setup.